src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
+
+{{ figure_markup(
+  image="websites_with_consent_management _platform.png",
+  caption="Websites using a consent management platform",
+  description="This figure shows the ahre of websites that are using a popular consent management platform.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=2025542332&format=interactive",
+  sheets_gid="1910033502",
+    sql_file="percent_of_websites_with_cmp.sql"
+  )
+}}
+{{ figure_markup(
+  image="consent_management_platform_popularity.png",
+  caption="Popularity of consent management platform",
+  description="This figure shows the popularity of each consent management platform.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=341496718&format=interactive",
+  sheets_gid="1104760876",
+    sql_file="percent_of_websites_using_each_cmp.sql"
+  )
+}}
+
+When looking at the popularity of the different consent management solutions, we can see that Osano and Quantcast choice are the leading platforms.
+
+#### IAB Europe’s Transparency Consent Framework
+IAB Europe proposed a [Transparency Consent Framework](https://iabeurope.eu/transparency-consent-framework/) (TCF) as a GDPR-compliant solution to obtain users' consent about their digital advertising preferences. The implementation provides an industry standard for communication between publishers and advertisers about consumer consent. 
+{{ figure_markup(
+  image="adoption_of_the_TCF_banner.png",
+  caption="Adoption rate of TCF banner",
+  description="This figure shows the percentage of websites using IAB Europe's TCF banner.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=341275612&format=interactive",
+  sheets_gid="2077755325",
+    sql_file="percent_of_websites_with_iab_tcf_banner.sql"
+  )
+}}
+While our results show that the TCF banner is not yet the ‘industry standard’, it is a step in the right direction. Considering the main target group of IAB Europe is in fact European publishers, having an adoption rate on 1.5% of websites on desktop client and 1.4% on mobile is not too bad. 
+
+## Privacy Policies
+Privacy policies are widely used by websites to increase transparency towards users about data collection practices. In our crawl, we searched for keywords indicating the presence of a privacy policy text on each visited website. 
```

Comment:
The cynic in me says this (and the cookie banner above) are not being done for transparency reasons!

```suggestion
Privacy policies are widely used by websites to met legal obligations and increase transparency towards users about data collection practices. In our crawl, we searched for keywords indicating the presence of a privacy policy text on each visited website. 
```

Reply:
@ydimova did you see this comment?